app/models/throttle.rb

```diff
@@ -18,7 +18,7 @@ class Throttle < ApplicationRecord
   THROTTLE_CONFIG = {
     idv_acuant: {
```

Comment:
Should the throttle name be updated as well? Fine to do separate if it balloons the branch.

Reply:
Great idea! Did that and renaming the max_attempts config param in 39eae3465